pkg/features/antrea_features.go

```diff
@@ -46,6 +46,10 @@ const (
 	// alpha: v0.9
 	// Flow exporter exports IPFIX flow records of Antrea flows seen in conntrack module.
 	FlowExporter featuregate.Feature = "FlowExporter"
+
+	// alpha: v0.10
+	// Adds IPv6 dual stack support for Antrea components.
+	IPv6DualStack featuregate.Feature = "IPv6DualStack"
```

Comment:
Since IPv6DualStack is also a FeatureGate in the upstreaming K8S when deploying the cluster, shall we share the same one? @tnqn is it possible to access the "FeatureGate" configured in the cluster from Antrea code?

Reply:
IDK if we could reuse the K8S featureGate.
Maybe instead of adding an Antrea featureGate, we could inquire if the cluster is running with IPv6DualStack enabled at init time, and act upon this as a global flag? IDK if this is doable either.

Reply:
Having synced with Quan offline, and got that it would be difficult to read the K8s cluster featuregates from APIs. So maybe we should start with Antrea's FeatureGates for `IPv6DualStack`